web/satellite/src/components/account/referral/ReferralArea.vue

```diff
@@ -0,0 +1,166 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+    <div class="referral-container">
+        <div class="referral-container__title-container">
+            <p class="referral-container__title-container__text">Refer A Friend And Help Build The</p>
+            <p class="referral-container__title-container__text">Decentralized Future</p>
+        </div>
+        <div class="referral-container__available" v-if="isAvailableLinks">
+            <p class="referral-container__available__title">You Have {{ 5 }} Invitations To Share!</p>
```

Comment:
The links are going to be dynamically generated right? Just double checking

Reply:
Sure it will

Reply:
Sure